modules/actions/utils.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package actions
+
+import (
+	"context"
+
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/models/db"
+	secret_model "code.gitea.io/gitea/models/secret"
```

Comment:
`modules` don't access the database.

Reply:
move to `service` package [14f2606](https://github.com/go-gitea/gitea/pull/29468/commits/14f2606023baf839ccd2a2bfc122291df2251ff0)